internal/handlers/common/parse.go

```diff
@@ -0,0 +1,42 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package common
+
+import (
+	"bufio"
+	"io"
+	"strings"
+
+	"github.com/FerretDB/FerretDB/internal/util/lazyerrors"
+)
+
+// ParseOSRelease parses the etc/os-release file.
+func ParseOSRelease(reader io.Reader) (map[string]string, error) {
```

Comment:
I think that function could return just name and version, so caller should not have to know what valid keys are

Reply:
Also, let's move it to msg_hostinfo.go and unexport

Reply:
Now `ParseOSRelease` return just name and version and moved to `msg_hostinfo.go` but in order to be able to use this function in tests, I left it exported